### PR TITLE
[luci/service] Support HardSwish operation

### PR DESCRIPTION
**Pull request by shs-park**
Support HardSwish operation in luci/service

ONE-DCO-1.0-Signed-off-by: Seungho Henry Park [shs.park@samsung.com](mailto:shs.park@samsung.com)

---

Related to: #10930 
Draft PR: #10985

### CODE REVIEW

**Comment by seanshpark** [comment]
We need to also update
- compiler/luci/service/include/luci/Service/CircleShapeInference.h
- compiler/luci/service/include/luci/Service/CircleTypeInference.h
- compiler/luci/service/src/CircleCloneNode.h

Please search with `CircleRelu` in `luci/service`

File: compiler/luci/service/src/Nodes/CircleHardSwish.test.cpp [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  auto node_hardswish = g->nodes()->create<luci::CircleHardSwish>();
+
+  auto gc = loco::make_graph();
+  auto cloned = luci::clone_node(node_hardswish, gc.get());
+  ASSERT_NE(nullptr, cloned);
+  ASSERT_EQ(gc.get(), cloned->graph());
+
+  auto cloned_hardswish = dynamic_cast<luci::CircleHardSwish *>(cloned);
+  ASSERT_NE(nullptr, cloned_hardswish);
+}
```
Comment (seanshpark) [inline compiler/luci/service/src/Nodes/CircleHardSwish.test.cpp:33]: Please refer `compiler/luci/service/src/Nodes/CircleRelu.test.cpp` and add shape and dtype inference test too.

Comment (shs-park) [reply compiler/luci/service/src/Nodes/CircleHardSwish.test.cpp:33]: Ok, I will check the file.
=)

Comment (shs-park) [reply compiler/luci/service/src/Nodes/CircleHardSwish.test.cpp:74]: I've added shape and dtype inference test for HardSwish.
- commit id: ccde26a

**Comment by shs-park** [comment]
> We need to also update
> 
> * compiler/luci/service/include/luci/Service/CircleShapeInference.h
> * compiler/luci/service/include/luci/Service/CircleTypeInference.h
> 
> Please search with `CircleRelu` in `luci/service`

Updated!
- commit id: ccde26a

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Pull request merged**